packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenuItemGroup.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+export interface OptionsMenuItemGroupProps extends React.HTMLProps<HTMLLIElement> {
+  /** Content to be rendered in the Options menu items component */
+  children?: React.ReactNode;
+  /** Classes applied to root element of the Options menu items group */
+  className?: string;
+  /** Provides an accessible name for the Options menu items group */
+  ariaLabel?: string;
```

Comment:
shouldn't this be 'aria-label'?  We should fix this in a future breaking change release to be consistent. @tlabaj

Reply:
See #1529 .